scan.sh

```diff
@@ -1,34 +1,49 @@
 #!/bin/sh
 
-echo "Initiating Fugue scan..."
-curl -s -X POST https://api.riskmanager.fugue.co/v0/scans?environment_id=$FUGUE_ENV_ID \
-   -u $FUGUE_CLIENT_ID:$FUGUE_CLIENT_SECRET | jq '.' > scaninfo.json 
+# Check for CLI env vars; print note about deprecated env vars
```

Comment:
I'm assuming a user won't set both sets of environment variables so for simplicity, I would write this whole block like:

```
if [ ! -z "$FUGUE_CLIENT_ID" ] || [ ! -z "$FUGUE_CLIENT_SECRET" ]; then
  printf "\nFUGUE_CLIENT_ID and FUGUE_CLIENT_SECRET are deprecated and will be removed in a future release. Please use FUGUE_API_ID and FUGUE_API_SECRET instead.\n"
  export FUGUE_API_ID=$FUGUE_CLIENT_ID
  export FUGUE_API_SECRET=$FUGUE_CLIENT_SECRET
fi

if [ -z "$FUGUE_API_ID" ] || [ -z "$FUGUE_API_SECRET" ]; then
  printf "\nError: No credentials detected. Please set the FUGUE_API_ID and FUGUE_API_SECRET credentials.\n"
  exit 1
fi

echo "Initiating Fugue scan..."
```

Example:

```
% FUGUE_CLIENT_ID=foo FUGUE_CLIENT_SECRET=bar ./becki.sh

FUGUE_CLIENT_ID and FUGUE_CLIENT_SECRET are deprecated and will be removed in a future release. Please use FUGUE_API_ID and FUGUE_API_SECRET instead.
Initiating Fugue scan...
```